### PR TITLE
Multiple closure functions

### PR DESCRIPTION
**Pull request by paulrolfe**
See the issue for more context if needed.

### CODE REVIEW

**Comment by loganwright** [comment]
#7 

:shipit:

File: README.md [hunk 1]
```diff
@@ -478,7 +478,32 @@ doSomethingWithCompletion() {
...
+    }
+)
+```
+**Not**
+
+(Even though the default spacing and syntax when you do this in xcode might end up looking like this)
+```swift
+UIView.animateWithDuration(SomeTimeValue, animations: {
+    // Do stuff
+    }) { _ in
```
Comment (brightredchilli) [inline README.md]: there is a syntax error here, no?

Comment (paulrolfe) [inline README.md]: Don't think so. What do you mean?

**Comment by loganwright** [comment]
I thought this had ben merged.  Ben added some stuff about closures in arguments as well, we may want to move this section to be nearby the other.

Then, let's get this merged!

:shipit:

**Comment by paulrolfe** [comment]
@Benuuu 's PR and I's are very similar. I think we should choose one.

**Comment by loganwright** [comment]
@paulrolfe I think there's a bit in each that's useful, maybe you could tag onto ben's and make the multi closure style a bit more explicit.

Share examples, but some of your reasoning I think could fit in

**Pull request merged**